charts/fluentd-elasticsearch/values.yaml

```diff
@@ -63,6 +63,9 @@ elasticsearch:
     enabled: false
     user: null
     password: null
+    existingSecret: null
```

Comment:
please remove null and comment empty values in.

Reply:
Sorted